mongodb/src/test/java/org/apache/calcite/adapter/mongodb/MongoAdapterTest.java

```diff
@@ -859,4 +859,26 @@ private static Consumer<List> mongoChecker(final String... expected) {
         .limit(2)
         .returns("STATE=VT; AVG(pop)=26408\nSTATE=AK; AVG(pop)=26856\n");
   }
+
+  @Test void testTranslateAndInCondition() {
```

Comment:
It may be necessary to include Jira's link and description on each test.

Reply:
@xiedeyantu 
Thank you for your suggestion. Done.